docs/eks-multipathd-fix.md

```diff
@@ -0,0 +1,19 @@
+## Installing Device Mapper Multipath on Amazon EKS
+
+If you are using EKS and need to use EBS and Pure Storage at the same time please read this blog.[Hey, Don't Break EBS](https://blog.2vcps.io/2020/06/16/hey-dont-break-ebs/)
```

Comment:
[nit] could we make the link be "this blog" instead? (Or if not, can we add a space between "blog." and "[Hey"?